src/cpp/src/text2image/stable_diffusion_pipeline.hpp

```diff
@@ -131,7 +131,6 @@ class Text2ImagePipeline::StableDiffusionPipeline : public Text2ImagePipeline::D
 
     void compile(const std::string& device, const ov::AnyMap& properties) override {
         m_clip_text_encoder->compile(device, properties);
-        m_unet->compile(device, properties);
```

Comment:
hello, just curious what the reason is to not compile unet here? Is the compile invoked someplace else now?

Reply:
looks like it's removed by mistake

Reply:
It was removed by mistake, reverted this change